test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -153,7 +154,15 @@ ava('.setDepartureRunway() throws when passed a string instead of a RunwayModel'
     t.throws(() => fms.setDepartureRunway(nextRunwayName));
 });
 
-ava.todo('.setDepartureRunway() validates route with changed runway');
+ava('.setDepartureRunway() regenerates SID legs for new runway', (t) => {
+    const nextRunwayMock = airportModelFixture.getRunway('19R');
+    const fms = buildFmsMockForDeparture();
+    const regenerateSidLegSpy = sinon.spy(fms, '_regenerateSidLeg');
+
+    fms.setDepartureRunway(nextRunwayMock);
+
+    t.true(regenerateSidLegSpy.called);
```

Comment:
sinon has methods `.calledWith()` and `.calledWithExactly()`.  I would encourage you to use one (ideally the latter) of those instead of `called`.

This allows us to not only test that a method is called, but also that it is called with the correct stuff.